src/molecules/advanced-calendar/index.tsx

```diff
@@ -29,33 +39,83 @@ const getLocale = () => {
   return allLocales[locale] || allLocales[rootLocale];
 };
 
+const dateOptions = [
+  {
+    value: 'today',
+    label: 'Today',
+  },
+  {
+    value: 'yesterday',
+    label: 'Yesterday',
+  },
+  {
+    value: 'this-week',
+    label: 'This Week',
+  },
+  {
+    value: 'last-week',
+    label: 'Last Week',
+  },
+  {
+    value: 'this-month',
+    label: 'This Month',
+  },
+  {
+    value: 'last-month',
+    label: 'Last Month',
+  },
+];
+
 export type AdvancedCalendarProps = CalendarProps & {
-  hideSelect?: boolean;
+  fromYear?: number;
```

Comment:
since you have deleted the hideSelect, did you check if it's used anywhere else in the project?

Reply:
Yes